packages/mobile/src/escrow/saga.ts

```diff
@@ -249,14 +266,25 @@ function* doFetchSentPayments() {
         timestamp: payment[6],
         expirySeconds: payment[7],
       }
-      sentPaymentsNotifications.push(transformedPayment)
+      sentPayments.push(escrowPaymentWithRecipient)
     }
-    yield put(storeSentPayments([...existingPayments, ...sentPaymentsNotifications]))
+
+    yield put(
+      storeSentEscrowPayments(forceRefresh ? sentPayments : [...existingPayments, ...sentPayments])
+    )
   } catch (e) {
     Logger.error(TAG + '@doFetchSentPayments', 'Error fetching sent escrowed payments', e)
   }
 }
 
+export function getReclaimableEscrowPayments(allPayments: EscrowedPayment[]) {
+  const currUnixTime = Date.now() / 1000
+  return allPayments.filter((payment) => {
+    const paymentExpiryTime = +payment.timestamp + +payment.expirySeconds
```

Comment:
Why the extra `+`?

Reply:
Not totally sure, I didn't write that code. Would rather leave it just in case